src/renderer.js

```diff
@@ -1,5 +1,7 @@
-var d3 = require('d3');
-var cartodb = cartodb || {};
+var d3 = window.d3 || require('d3');
```

Comment:
Not so important but all these should be `global` instead of `window`. See https://github.com/substack/browserify-handbook#global